app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -277,6 +250,18 @@
                     app:menu="@menu/multiselection_menu"
                     tools:visibility="visible" />
 
+                <com.infomaniak.mail.ui.main.EmptyStateView
+                    android:id="@+id/emptyStateView"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:visibility="gone"
+                    app:layout_anchor="@+id/swipeRefreshLayout"
+                    app:layout_anchorGravity="center"
+                    tools:description="@string/emptyStateInboxDescription"
+                    tools:icon="@drawable/ic_empty_state_folder"
+                    tools:title="@string/emptyStateInboxTitle"
+                    tools:visibility="visible" />
+
```

Comment:
Why did you need to move the EmptyStateView outside the FrameLayout?

Reply:
Not sure how it happened. Thanks.